libs/sdk-backend-base/src/normalizingBackend/normalizer.ts

```diff
@@ -117,14 +119,16 @@ export class Denormalizer {
             } else if (isMeasureDescriptor(value)) {
                 const localIdentifier = this.originalLocalId(value.measureHeaderItem.localIdentifier);
                 const measure = this.originalMeasures[localIdentifier]!;
-                const format = measureFormat(measure) || value.measureHeaderItem.format;
+                const format = isDerivedMeasure(measure)
+                    ? this.getDerivedMeasureFormat(measure)
+                    : measureFormat(measure);
```

Comment:
pls wrap this in separate function, use similar method to how measure titles are provided.
function takes measure + the fallback to server-provided value. returns value to use.